code/modules/faction/faction_datum.dm

```diff
@@ -79,3 +97,9 @@
 	name = FACTION_INDEPENDENT
 	short_name = "Indie"
 	prefixes = PREFIX_INDEPENDENT
+
+/datum/faction/scarborough_arms
```

Comment:
Any reason not to just outright make it `/datum/faction/syndicate/scarborough_arms`?

Reply:
yeah no clue why i didnt, it should now